src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/stash/StashApiClient.java

```diff
@@ -172,30 +177,16 @@ public StashPullRequestMergeableResponse getPullRequestMergeStatus(String pullRe
 
   public boolean mergePullRequest(String pullRequestId, String version) {
     String path = pullRequestPath(pullRequestId) + "/merge?version=" + version;
-    try {
-      String response = postRequest(path, null);
-      return !response.equals(Integer.toString(HttpStatus.SC_CONFLICT));
-
-    } catch (UnsupportedEncodingException e) {
-      e.printStackTrace();
-    } catch (IOException e) {
-      logger.log(Level.SEVERE, "Failed to merge Stash PR " + path + " " + e);
-    }
-    return false;
+    String response = postRequest(path, null);
+    return !response.equals(Integer.toString(HttpStatus.SC_CONFLICT));
   }
 
   private HttpClient getHttpClient() {
     HttpClient client = new HttpClient();
     HttpParams httpParams = client.getParams();
-    // ConnectionTimeout : This denotes the time elapsed before the connection established or Server
-    // responded to connection request.
     httpParams.setParameter(
-        CoreConnectionPNames.CONNECTION_TIMEOUT,
-        StashApiClient.HTTP_CONNECTION_TIMEOUT_SECONDS * 1000);
-    // SoTimeout : Maximum period inactivity between two consecutive data packets arriving at client
-    // side after connection is established.
-    httpParams.setParameter(
-        CoreConnectionPNames.SO_TIMEOUT, StashApiClient.HTTP_SOCKET_TIMEOUT_SECONDS * 1000);
+        CoreConnectionPNames.CONNECTION_TIMEOUT, HTTP_CONNECTION_TIMEOUT_SECONDS * 1000);
+    httpParams.setParameter(CoreConnectionPNames.SO_TIMEOUT, HTTP_SOCKET_TIMEOUT_SECONDS * 1000);
```

Comment:
I guess using `TimeUnit` would be better than just `* 1000` but it's not a big deal

Reply:
All timeouts are in milliseconds both in the old and in the new HTTP clients. I cannot force the HTTP client to take seconds and convert them for us.

I can specify all the timeouts in milliseconds if you want, but that would be a separate PR.